pkg/controller/pxc/users.go

```diff
@@ -455,6 +468,23 @@ func (r *ReconcilePerconaXtraDBCluster) manageOperatorAdminUser(cr *api.PerconaX
 	return nil
 }
 
+func (r *ReconcilePerconaXtraDBCluster) manageReplicationUser(cr *api.PerconaXtraDBCluster, sysUsersSecretObj *corev1.Secret) error {
```

Comment:
I think we should create a user only if it doesn't exist
```
	pass, existInSys := sysUsersSecretObj.Data["replication"]
	_, existInInternal := internalSysSecretObj.Data["replication"]
	if existInSys && !existInInternal {
		if internalSysSecretObj.Data == nil {
			internalSysSecretObj.Data = make(map[string][]byte)
		}
		internalSysSecretObj.Data["replication"] = pass
		return nil
	}
	if existInSys {
		return nil
	}
```

Reply:
it will not work the same, coz operator user created from entrypoint sh file

Reply:
so I think we should create it also from entrypoint sh file for new setups